dipti2jcommerce/magento2
# [Backport] Fixed Issue #20234

### Original Pull Request 
 https://github.com/magento/magento2/pull/22694




### Description (*)

Review text Should be capitalized

### Fixed Issues (if relevant)

1. magento/magento2#<20234>: Review text Should be capitalized

### Manual testing scenarios (*)
1. Navigate to product listing page.
2. Change the mode from grid to list.
3. Now you can see "add your review"  and "review" text in this format 
   Add Your Review
   Review

### Contribution checklist (*)
